doc/_sphinx/theme/doctools.js

```diff
@@ -72,12 +79,12 @@ const _highlight = (node, addItems, text, className, index) => {
       }
     }
   } else if (node.matches && !node.matches("button, select, textarea")) {
-    node.childNodes.forEach((el) => _highlight(el, addItems, text, className, index));
+    node.childNodes.forEach((el) => _highlight_flame(el, addItems, text, className, index));
   }
 };
-const _highlightText = (thisNode, text, className, index) => {
+const _highlightText_flame = (thisNode, text, className, index) => {
```

Comment:
This should be changed too right?
```suggestion
const _highlightTextFlame = (thisNode, text, className, index) => {
```

Reply:
Sure.  Now that it doesn't reference the theme anymore, I don't think it matters.